pipenv/project.py

```diff
@@ -801,7 +801,11 @@ def get_lockfile_hash(self):
         if not os.path.exists(self.lockfile_location):
             return
 
-        lockfile = self.load_lockfile(expand_env_vars=False)
+        try:
+            lockfile = self.load_lockfile(expand_env_vars=False)
+        except json.decoder.JSONDecodeError:
+            # Lockfile corrupted
+            return
```

Comment:
Better to return an empty string here. A lockfile without hash entry can be treated as corrupt as well.

Also `JSONDecodeError` was added in 3.5, and not available on 2.7. Catch `ValueError` instead.